### ISSUE TITLE
Dictionary is only JSONEncodable if Value is (and if Key is String)

### ISSUE THREAD

**Issue by hr147** [apollographql/apollo-ios#150]
I updated to 0.6.5 and started having this strange issue with apollo 0.6.5. It was working fine in apollo 0.5.3
![screen shot 2017-09-22 at 12 06 48 pm](https://user-images.githubusercontent.com/15008942/30733416-454a0f34-9f90-11e7-9247-588dfa46a039.png)

```
query Feed($user_id: ID!){
  allPosts(
    orderBy: id_DESC,
    first: 5,
    skip: 0,
    filter: {
        user: {
            followerUsers_some: {
                id: $user_id
            }
        }
    }
  ){
    id
    type
    description
    user{
        id
        firstName
        lastName
        profilePicture
    }
  }
 
}
```

**Comment by martijnwalraven** [comment]
Sorry, but when I asked you to post your query on the issue, I meant the existing issue, not a new one. Closing this as a duplicate of #127, so we can continue the discussion there.

**Comment by martijnwalraven** [comment]
@hr147: This should be fixed in Apollo iOS 0.7.0-alpha.6.
